src/Bridge/Symfony/Bundle/Resources/config/metadata.xml

```diff
@@ -116,24 +116,12 @@
 
         <!-- Cache -->
 
-        <service id="api_platform.metadata.resource.cache.array" class="Symfony\Component\Cache\Adapter\ArrayAdapter" public="false">
-            <argument>0</argument>
-            <argument>false</argument>
+        <service id="api_platform.metadata.resource.cache" parent="cache.system" public="false">
```

Comment:
This has nothing to do with Symfony. As far as Symfony is concerned, everything Symfony goes under the "system" cache pool.

In any case, I'm just trying to clarify the semantics, and my conclusion seems to be consistent with yours (at least in this case).